### ISSUE TITLE
Discord bridge: "Nightwatch" confuses people

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K-Infrastructure#2415]
Every now and then we get people who are confused about Nightwatch. It could be called some name that conveys "ingame lobby bridge". This doesn't require touching the account credentials itself so should be harmless and doable using just its login and password (simple DB task?).

**Comment by Licho1** [comment]
its hardcoded in many places, including discord bot itself and other stuff. Its not as harmless as you think.
Possibly its even in lua ingame.

**Comment by sprunk** [comment]
This is just about the discord "visible" name. Its "backend" account credential name, ZK lobby account name etc would stay "Nightwatch".
![nightwatch](https://user-images.githubusercontent.com/2573076/45877740-28fe3c00-bd9f-11e8-8d0e-1c5310984d45.png)

**Comment by Licho1** [comment]
Yeah but when it gets message from discord, it compares it to nightwatch too afaik to decide whether to relay or not.
